packages/visualizations/src/components/Map/Combinations/MiniMap.svelte

```diff
@@ -0,0 +1,38 @@
+<script lang="ts">
+    import SvgChoropleth from '../Svg/Choropleth.svelte';
+    import type { ColorScale } from '../../types';
+    import type { ChoroplethDataValue, NavigationMap } from '../types';
+    import Tooltip from './Tooltip.svelte';
+
+    export let active: boolean;
+    export let showTooltip: boolean;
+    export let data: { value: ChoroplethDataValue[] };
+    export let map: NavigationMap;
+    export let colorScale: ColorScale | undefined;
+
+    $: ({ label } = map);
+    $: options = { shapes: map.shapes, colorScale };
+</script>
+
+{#if showTooltip}
+    <Tooltip {label}>
```

Comment:
In React we could have mutualized the button code in a separate component declared inline the same file, so that it would look like
```svelte
{#if showTooltip}
    <Tooltip {label}>
        {minimapButton}
    </Tooltip>
{:else}
    {minimapButton}
{/if}
```

Is there a clean way to do something similar in Svelte (without creating an entire separate component file?)

It would be a bit cleaner and prevent future mistakes where we edit one but not the other, but maybe not at the price of the overhead of an entire separate component...

Reply:
As far as I know, it has to be a separate component. But we shouldn't be afraid of doing a lot of small components IMO. I can do that!